tests/test.sh

```diff
@@ -32,7 +32,7 @@ expect() {
 
     echo "$expected_output" > "$tmp_expected"
 
-    "$fd" "$@" | sort -f > "$tmp_output"
+    "$fd" "$@" | xargs -0 echo -n | sort -f > "$tmp_output"
```

Comment:
Oh, I see. I thought about somehow adding the two characters `\0` in the shell script (not the null-character itself), like in `echo -e "a\0b"`.